ledger/store/trackerdb/sqliteImpl/trackerdbV2.go

```diff
@@ -400,26 +381,14 @@ func (tu *trackerDBSchemaInitializer) deleteUnfinishedCatchpoint(ctx context.Con
 	}
 
 	relCatchpointFilePath := filepath.Join(
-		CatchpointDirName,
-		MakeCatchpointFilePath(basics.Round(round)))
-	err = RemoveSingleCatchpointFileFromDisk(tu.DbPathPrefix, relCatchpointFilePath)
+		trackerdb.CatchpointDirName,
```

Comment:
looks weird, the child package refers to parent, usually it is done another way around